### PR TITLE
Mark workspace as safe in all actions

### PR DESCRIPTION
**Pull request by fg-j**
## Use Cases

Addresses automation failures like: https://github.com/paketo-buildpacks/full-builder/runs/6011835387?check_suite_focus=true

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by sophiewigmore** [comment]
@fg-j I just missed 'em when I did a grep. I can add them now. Thanks!!

**Comment by fg-j** [comment]
@paketo-buildpacks/tooling-maintainers I can't approve this bc I own the PR.

**Review by ryanmoran — APPROVED** [review]

**Pull request merged**
